app/serializers/curator/filestreams/audio_serializer.rb

```diff
@@ -2,15 +2,5 @@
 
 module Curator
   class Filestreams::AudioSerializer < Filestreams::FileSetSerializer
-    schema_as_json do
```

Comment:
Is this stuff going to be replaced at some point?

Reply:
I took it out since its not needed anymore in regards to the avi_processor